facets_mcp/utils/intent_utils.py

```
@@ -0,0 +1,70 @@
+import os
+import yaml
+from typing import Tuple
+from facets_mcp.utils.client_utils import ClientUtils
+from swagger_client.api.intent_management_api import IntentManagementApi
+from swagger_client.rest import ApiException
+
+def check_intent_and_intent_details(module_path: str) -> Tuple[bool, str]:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)